### PR TITLE
Initial Draft of the Delivery WG Charter

### PR DESCRIPTION
**Pull request by damienjburks**
Collaborating on this with @eddie-knight, @mlysaght2017, and @robmoffat.

Part of #171

### CODE REVIEW

File: docs/governance/working-groups/delivery/charter.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Delivery Working Group
+
+This [WG] defines and maintains the delivery process for the Common Cloud Controls project deliverables.
+
+## Mission
+
+The mission of the Delivery [WG] is to establish robust, scalable, and efficient frameworks for the governance and operation of all community [WG]s within the CCC.
+
+By designing clear guidelines and structures, this group ensures consistency and facilitates effective collaboration across the entire community. It acts as a central body to guide the formation and operation of other [WG]s, helping them align with the overarching goals of the CCC.
```
Comment (rowan-baker) [inline docs/governance/working-groups/delivery/charter.md]: This is from the community charter WG mission. Just to highlight that this should be changed/removed.

Comment (damienjburks) [reply docs/governance/working-groups/delivery/charter.md]: It was updated in this commit: https://github.com/finos/common-cloud-controls/pull/167/commits/0019b73bad95b0b23a871087c0cf17c781a1f4bc

Let me know if you have any more concerns.

**Review by AdrianHammond — COMMENTED** [review]
+1 LGTM

**Review by AdrianHammond — COMMENTED** [review]
@eddie-knight  - your are SC Sponsor on a number of WG, do you need other members of the SC to take on some of these responsibilities?

**Comment by eddie-knight** [comment]
Yes @AdrianHammond, that would help!

**Review by pgste — COMMENTED** [review]
+1 - Approved

**Comment by jonmuk** [comment]
+1

**Comment by rgriffiths-scottlogic** [comment]
+1 - Approved

**Comment by eddie-knight** [comment]
+1

**Review by rgriffiths-scottlogic — APPROVED** [review]
Approve

**Pull request merged**